frontend/src/components/Volunteer/volunteer.jsx

```diff
@@ -8,195 +8,68 @@ class Volunteer extends Component {
     super(props);
 
     this.state = {
-      role: "",
-      name: "",
-      email: "",
-      phone: "",
-      website: "",
-      message: ""
+      message: "",
+      loadCounter: 0
     };
   }
 
-  onChange = event => {
-    const target = event.target;
-    const value = target.value;
-    const name = target.name;
-
-    this.setState({
-      [name]: value
-    });
-  };
-
-  onFormSubmit = e => {
-    e.preventDefault();
-
-    console.log("data is being sent", this.state);
-    // fetch("http://the backend", {
-    //   method: "POST",
-    //   body: JSON.stringify(this.state),
-    //   headers: { "Content-Type": "application/json" }
-    // }).then(function() {
-    //   console.log("data has been set");
-    // });
-
-    this.setState({
-      role: "",
-      name: "",
-      email: "",
-      phone: "",
-      website: "",
-      message: ""
-    });
+  loaded = () => {
+    this.state.loadCounter += 1;
+    if (this.state.loadCounter % 2 === 0)
```

Comment:
what is the purpose of this?

Reply:
Thank you for the review.

This is a good question. This code snippet is used to toggle between two heights, the larger height to accommodate the large empty Google form and the smaller to adjust the screen to the shorter Google form submission confirmation window. Without out it, the user will see a large whitespace after they submit their form entry. This method is invoked in line 39 within the iframe. To be honest, I don't understand this workaround in depth but it works.

Reply:
The validation of the phone number for `( )` was requested by @chitownhope. The result is a nice consistent format in the resulting Google sheet for our client. Please see column E in the attached screenshot. 

![Screenshot from 2019-10-16 14-24-58](https://user-images.githubusercontent.com/29875882/66952800-eac76080-f022-11e9-9336-b332e5fdd7b6.png)

Reply:
I forgot to comment earlier that I changed one small part of the client's text. In the third paragraph, it had stated "In the form below, ...". I thought this was inaccurate since the form appears to the left while in wide mode and beneath while in mobile/tablet mode. So I changed it to "In the accompanying form, ...".

Reply:
Ok I looked more into this and all this is doing is setting the height. We can just replace this by setting the height in the css. and not need this weird workaround

```
.resp-iframe {
  height: 1050px;
}
```